vite.config.js

```diff
@@ -119,7 +121,7 @@ export default ({ mode }) => {
       extensions: allExtensions,
       alias: [
         {
-          find: /^((api|cheatcodes|features|fixtures|libs|queries|shared|theme|ui|web).*)/, // if you change this line, check this doc https://github.com/pass-culture/pass-culture-app-native/blob/5ff5fba596244a759d60f8c9cdb67d56ac86a1a7/doc/development/alias.md
+          find: /^((api|cheatcodes|features|fixtures|libs|queries|shared|theme|ui|web|tests).*)/,
```

Comment:
j'aimerais conserver le message d'aide pour maintenir la conf synchro partout

```suggestion
          find: /^((api|cheatcodes|features|fixtures|libs|queries|shared|theme|ui|web|tests).*)/, // if you change this line, check this doc https://github.com/pass-culture/pass-culture-app-native/blob/5ff5fba596244a759d60f8c9cdb67d56ac86a1a7/doc/development/alias.md
```